### ISSUE TITLE
bug: MessageNotification component does not jump to bottom when clicked after scrolling too far up (to load new messages)

### ISSUE THREAD

**Issue by Briiick** [GetStream/stream-chat-react#1687]
**Describe the bug**

When implementing a "new messages!" notification button, the button only jumps to the most recent messages in chat when the user hasn't scrolled too high. If older messages load in the chat box, the button remains but no longer works upon click (nothing happens).

Likely related to the `scrollToBottom` logic in https://github.com/GetStream/stream-chat-react/blob/master/src/components/MessageList/MessageList.tsx

**To Reproduce**

1. Set up `MessageNotification` button in functioning stream application with a lot of messages.
```typescript
<Channel
          channel={channel}
          MessageNotification={MessageNotification}
>
```
2. Try clicking the button if you scroll up a little bit and post new messages with separate user account (should work)
3. Try clicking the button if you scroll up a lot and load old messages further up the chat with separate user account (won't work).

**Expected behavior**

Regardless of the amount the user scrolls up, as soon as a new message is posted by a different user, the button should pop up and be clickable.

**Screenshots**

https://www.loom.com/share/ed1f728952534acfab43abee6ef13f12

**Package version**
- stream-chat-react: ^9.1.5
- stream-chat: ^6.7.3

**Desktop (please complete the following information):**

- Browser: Chrome
- Version: 103.0.5060.134

**Comment by zencephalon** [comment]
I've also run into this behavior!

**Comment by MartinCupela** [comment]
Thank you for reporting this @Briiick & @zencephalon! We are taking look at the issue.

**Comment by MartinCupela** [comment]
fixed by #1689
